lambdas/build-cri-oauth-request/src/test/java/uk/gov/di/ipv/core/buildcrioauthrequest/BuildCriOauthRequestHandlerTest.java

```diff
@@ -775,22 +786,23 @@ private void assertSharedClaimsJWTIsValid(String request)
             addressList.add(OBJECT_MAPPER.convertValue(jo, Address.class));
         }
 
+        // Find the latest Address VC (from which we take the address)
         Address streetAddress =
                 addressList.stream()
                         .filter(x -> "NotDowningStreet".equals(x.getStreetName()))
                         .findAny()
                         .orElse(null);
         Address postCode =
                 addressList.stream()
-                        .filter(x -> "SW1A2AA".equals(x.getPostalCode()))
+                        .filter(x -> "SW2A 3BB".equals(x.getPostalCode()))
                         .findAny()
                         .orElse(null);
 
-        assertFalse(streetAddress.getStreetName().isEmpty());
-        assertFalse(postCode.getPostalCode().isEmpty());
+        assertNotNull(streetAddress);
+        assertNotNull(postCode);
 
         assertEquals(2, (vcAttributes.get("name")).size());
-        assertEquals(3, (vcAttributes.get("address")).size());
+        assertEquals(1, (vcAttributes.get("address")).size());
```

Comment:
We explicitly have code that means we only take the address from one address VC! (I'm not 100% sure that's correct, but I'm inclined to leave the implementation as-is for now)